### PR TITLE
Add an option for using the real API endpoint

### PR DESCRIPTION
**Pull request by inopia**

### CODE REVIEW

File: procountor/client.py [hunk 1]
```diff
@@ -66,7 +78,7 @@ def get_auth_code(self):
             'content-type': 'application/x-www-form-urlencoded'
         }
 
-        r = requests.post(Client.auth_url + '?response_type=code&client_id=' + self.client_id + "&state=123456",
+        r = requests.post(self.api_url + 'oauth/authz/?response_type=code&client_id=' + self.client_id + "&state=123456",
```
Comment (Jormatosijorma) [inline procountor/client.py]: line 81 is too long (max 120 characters) and failed flake8 test

Comment (Jormatosijorma) [inline procountor/client.py]: line 81 is too long (max 120 characters) and failed flake8 test

**Review by Jormatosijorma — CHANGES_REQUESTED** [review]

**Comment by inopia** [comment]
Commit updated

**Review by Jormatosijorma — APPROVED** [review]

**Comment by Jormatosijorma** [comment]
Tested this locally 'cause automatic tests doesn't work due to secure environment variables

**Pull request merged**